app/views/project_features/new.html.erb

```diff
@@ -34,7 +34,7 @@
         </div>
       </div>
     </div>
-  <div>
+  </div>
```

Comment:
sửa chỗ này là cái nút submit bên dưới nó bị to lắm đấy tiện fix luôn nhá

Reply:
À vâng thế thì thôi a :)), e thấy ide nó warning nên sửa thôi